api-inference-community/docker_images/speechbrain/app/pipelines/audio_to_audio.py

```diff
@@ -0,0 +1,61 @@
+from typing import Tuple, List
+
+import numpy as np
+import torch
+from app.pipelines import Pipeline
+from speechbrain.pretrained import SepformerSeparation, SpectralMaskEnhancement
+from huggingface_hub import HfApi
+
+
+class AudioToAudioPipeline(Pipeline):
+    def __init__(self, model_id: str):
+        info = HfApi().model_info(model_id)
+        tags = [tag.lower().replace(" ", "-").replace("_", "-") for tag in info.tags]
```

Comment:
imo we should not do this, this will only lead to different tags "Audio Source Separation" and "audio-source-separation" and users will get confused. There are only 4 repos with tag="Audio Source Separation" and 2 for "Speech Enhancement"

* https://huggingface.co/models?filter=Audio%20Source%20Separation
* https://huggingface.co/models?filter=Speech%20Enhancement

Why not go through them and change to `audio-source-separation` or `speech-enhancement`?

Reply:
Fair enough. Was practical to test right away.

Less tricks like this = less suprise, but sometimes it's less convenient.

Ok with being stricter.

Reply:
Done.